### PR TITLE
Add pages for most of the missing documentation

### PR DESCRIPTION
**Pull request by dkanada**

### CODE REVIEW

**Comment by anthonylavado** [comment]
Oops plugin change again. I would have merged this already but I forgot.

File: docs/administrator-docs/hardware-acceleration.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
-# Hardware Acceleration in Jellyfin
+# Hardware Acceleration
 
 Jellyfin supports hardware acceleration of video encoding/decoding/transcoding using FFMpeg. It supports multiple acceleration types, including  AMD AMF, Intel Quick Sync, OpenMax OMX, nVidia NVENC, Intel/AMD VAAPI, and others.
 
-## Enabling hardware acceleration
+## Enabling Hardware Acceleration
 
-Hardware acceleration options can be found in the Admin Dashboard under the Transcoding section. Select a valid "Hardware acceleration" from the drop-down, indicate a device if applicable, and, if supported, "Enable hardware encoding" as well as decoding.
+Hardware acceleration options can be found in the Admin Dashboard under the **Transcoding** section. Select a valid hardware acceleration option from the drop-down menu, indicate a device if applicable, and check `enable hardware encoding` as well as decoding if your hardware supports that as well.
```
Comment (joshuaboniface) [inline docs/administrator-docs/hardware-acceleration.md]:
```suggestion
Hardware acceleration options can be found in the Admin Dashboard under the **Transcoding** section. Select a valid hardware acceleration option from the drop-down menu, indicate a device if applicable, and check `enable hardware encoding` to enable encoding as well as decoding, if your hardware supports this.
```

File: docs/server/transcoding.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+## Transcoding
+
+These settings will relate to backend options that modify how the server transcodes media. Some improve or change the media quality while others reduce the resources required to transcode the media from its original format.
+
+#### Hardware Acceleration
+
+If your hardware supports this you can enable hardware acceleration for much faster transcoding. Some of the supported methods are listed below.
```
Comment (joshuaboniface) [inline docs/server/transcoding.md]:
```suggestion
 If your hardware supports this you can enable [hardware acceleration](/administrator-docs/hardware-acceleration) for much faster transcoding. Some of the supported methods are listed below.
```

**Comment by joshuaboniface** [comment]
Once that conflict is fixed up, we'll be good to go cc @dkanada.

**Pull request merged**